Shared/Samples/Create convex hull around geometries/CreateConvexHullAroundGeometriesView.swift

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct CreateConvexHullAroundGeometriesView: View {
+    /// A map with a topographic basemap.
+    @State private var map = Map(basemapStyle: .arcGISTopographic)
+    
+    /// The graphics overlay for the geometry graphics.
+    @State private var geometriesGraphicsOverlay: GraphicsOverlay = {
+        let polygonFillSymbol = SimpleFillSymbol(
+            style: .forwardDiagonal,
+            color: .systemBlue,
+            outline: SimpleLineSymbol(style: .solid, color: .blue, width: 2)
+        )
+        
+        // Create graphics for the example polygons.
+        let polygonGraphic1 = Graphic(geometry: .polygon1, symbol: polygonFillSymbol)
+        let polygonGraphic2 = Graphic(geometry: .polygon2, symbol: polygonFillSymbol)
+        return GraphicsOverlay(graphics: [polygonGraphic1, polygonGraphic2])
+    }()
+    
+    /// The graphics overlay for the convex hull graphics.
+    @State private var convexHullGraphicsOverlay = GraphicsOverlay()
+    
+    /// A Boolean indicating whether the button is set to create.
+    @State private var createIsOn = true
+    
+    /// A Boolean indicating whether the convex hull geometries should union.
+    @State private var shouldUnion = false
+    
+    var body: some View {
+        MapView(map: map, graphicsOverlays: [convexHullGraphicsOverlay, geometriesGraphicsOverlay])
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Toggle("Union", isOn: $shouldUnion)
+                        .toggleStyle(.switch)
+                        .onChange(of: shouldUnion) { _ in
+                            createConvexHulls()
+                        }
+                    Button(createIsOn ? "Create" : "Reset") {
+                        if createIsOn {
+                            createConvexHulls()
+                            createIsOn = false
+                        } else {
+                            convexHullGraphicsOverlay.removeAllGraphics()
+                            createIsOn = true
+                        }
+                    }
+                }
+            }
+    }
+}
+
+private extension CreateConvexHullAroundGeometriesView {
+    /// Creates convex hulls from the example polygons.
+    func createConvexHulls() {
```

Comment:
Better making this a pure function `func makeConvexHulls(for geometries: [Geometry], unioned: Bool) -> [Geometry]` so it is easier for users to copy and paste. And handle the graphic overlay add/delete outside the method.